lib/between_meals/repo/git.rb

```diff
@@ -168,7 +173,7 @@ def parse_status(changes)
         # X: "unknown" change type (most probably a bug, please report it)
 
         # rubocop:disable MultilineBlockChain
-        changes.lines.map do |line|
+        changes.lines.to_a.reverse.map do |line|
```

Comment:
To handle the case of a path being removed and re-added as a submodule.  Before reversing the order it was only picking up the delete, and not the additions.